### ISSUE TITLE
[Bug]: Inline Node View - text gets trapped inside node

### ISSUE THREAD

**Issue by fbolcic** [ueberdosis/tiptap#4522]
### Which packages did you experience the bug in?

core, react

### What Tiptap version are you using?

2.1.11

### What’s the bug you are facing?

I am trying to add an inline node view. I am following the example from [the docs](https://tiptap.dev/guide/node-views/react) with the exception of creating an inline node (as opposed to block).

I can create the node with no difficulty. The problem I am facing surfaces when I try to edit the text inside the node or immediately after. Somehow the text "gets stuck" inside the node view wrapper and text editing cannot be continued normally. The issue can best be seen on the following video (example is from the provided code sandbox):

https://github.com/ueberdosis/tiptap/assets/101173923/66eb4d10-6364-4eba-9683-b922148285b3

Here I am initialising the editor with my custom node view (`a` tag). You can see that the text added immediately after the node gets stuck and editing cannot be continued normally.

### What browser are you using?

Chrome

### Code example

https://codesandbox.io/s/zen-frog-ptyrp8?file=/src/NodeView.tsx

### What did you expect to happen?

I expect to continue editing normally. I should be able to add text after the node without it getting stuck.

### Did you update your dependencies?

- [X] Yes, I’ve updated my dependencies to use the latest version of all packages.

### Are you sponsoring us?

- [ ] Yes, I’m a sponsor. 💖

**Comment by joshuaR-G** [comment]
I'm also facing the same issue, did you find a way to deal with this? @fbolcic

**Comment by fbolcic** [comment]
No, we deprioritised the issue :/. I was hoping I would get some feedback from the maintainers but no luck 🤷 .

**Comment by fbolcic** [comment]
> I'm also facing the same issue, did you find a way to deal with this? @fbolcic

No, we deprioritised the issue :/. I was hoping I would get some feedback from the maintainers but no luck 🤷 .
Let me know if you make some advancement.

**Comment by joshuaR-G** [comment]
> > I'm also facing the same issue, did you find a way to deal with this? @fbolcic
> 
> No, we deprioritised the issue :/. I was hoping I would get some feedback from the maintainers but no luck 🤷 . Let me know if you make some advancement.

Damn.. not sure if it's a valid workaround yet but removing the NodeViewContent tag altogether seemed to fix the immediate issue of cursor trapping for me.

Just using the <NodeViewWrapper /> allowed for the cursor to move in and out of the Node.

**Comment by JoelHutchinson** [comment]
Setting atom=true seemed to fix the issue for me.

**Comment by GhostyCHEN** [comment]
I'm also facing the same issue,if someone find any suggestion please let me know!
